src/components/Recipes/RecipeList.js

```diff
@@ -1,18 +1,18 @@
 import React from 'react'
+import styled from 'styled-components'
 import recipes from '../__mocks__/recipes.json'
 import RecipeListItem from './RecipeListItem'
-import styled from 'styled-components'
 
 export default function RecipeList() {
   return (
-    <StyledRecipeList>
+    <StyledRecipeList id="recipes">
       {recipes.map(({ id, title }) => (
-        <RecipeListItem key={id} title={title} />
+        <RecipeListItem id={id} key={id} title={title} />
```

Comment:
Schau mal, ob du was anderes als die id als key hier verwendest. Ich glaube, id's sollte man nicht unbedingt verwenden, ich vermute, dass sie dich Reihe der List-Items häufiger mal bei dir ändert oder? Falls nicht mein Kommentar ignorieren :)
`We don’t recommend using indexes for keys if the order of items may change. `
https://reactjs.org/docs/lists-and-keys.html

Reply:
Kann es sein, dass du indexes statt id meinst? Die id wird für jedes item individuell vergeben und dürfte sich deshalb auch bei Änderung der Reihenfolge nicht verändern.